draft-ietf-httpbis-http2bis.xml

```diff
@@ -4383,7 +4383,7 @@ cookie: e=f
       <section anchor="iana-h2c">
         <name>The h2c Upgrade Token</name>
         <t>
-         Previous versions of this document (<xref target="RFC7540" section="11.8"/>) registered an upgrade
+         <xref target="RFC7540" section="11.8">RFC7540</xref> registered an upgrade
```

Comment:
```suggestion
         <xref target="RFC7540" section="11.8"/> registered an upgrade
```
I don't know what your version would have rendered as.  If you want to try it out and share, then maybe try that.  With a space, of course.

Reply:
I don’t have a proper computer on me and I’m too lazy to spin up a cloud VM so I’m just gonna do it your way.